docs/allow-nested-if-parent-exist.md

```diff
@@ -0,0 +1,56 @@
+# AllowNestedIfParentExist
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-11
+
+## Description
+
+When the parent resource is scoped under a resource group, the path for the list calls of its child resources must also be scoped under a resource group
+
+## How to fix
+
+To fix this, ensure that the LIST call for the child resource has a resource group segment.
+
+### Invalid example
+
+Following are the patterns for the invalid examples
+GET /subscriptions/{sub1}/providers/{ProviderName}/resourceType1/{resourceTypeName}/nestedResourceType
+GET /subscriptions/{sub1}/providers/{ProviderName}/nestedResourceType
+GET /subscriptions/{sub1}/resourcegroups/{rgname}/providers/{ProviderName}/nestedResourceType
+
+```json
+"paths": {
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ProviderNamespace/ParentResourceType/{parentInstance}/NestedResourceType": {
+    "get": {
+        //Nested Resource Type Get Call definition
+    },
+  }
+// No Parent Resource Get Call definied
+}
+```
+
+### Valid example
+
+```json
+"paths": {
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ProviderNamespace/ParentResourceType/{parentInstance}": {
+    "get": {
+      //Parent Resource Get Call definition
+    },
+  },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ProviderNamespace/ParentResourceType/{parentInstance}/NestedResourceType": {
+    "get": {
+        //Nested Resource Type Get Call definition
+    },
+  }
+}
+```
```

Comment:
Cannot add in the deleted files, but aint those deletes already taken care in your other PR?

Reply:
After merging the PR https://github.com/Azure/azure-openapi-validator/pull/571, now everything got in sync.
And we dont see changes related to this rule "RPC-Put-V1-11" in this PR